src/doc/src/reference/specifying-dependencies.md

```diff
@@ -19,42 +19,44 @@ guide](../guide/index.md), we specified a dependency on the `time` crate:
 time = "0.1.12"
 ```
 
-The string `"0.1.12"` is a [semver] version requirement. Since this
-string does not have any operators in it, it is interpreted the same way as
-if we had specified `"^0.1.12"`, which is called a caret requirement.
-
-[semver]: https://github.com/steveklabnik/semver#requirements
```

Comment:
This link is now dead, and the new docs actually refer to *this* document to explain how requirements work :rofl: